lms/static/sass/course/courseware/_courseware.scss

```diff
@@ -240,6 +244,10 @@ div.course-wrapper {
         padding: 0;
       }
     }
+
+    @media print {
```

Comment:
Removes extra whitespace from around the content. The page that's being printed on already has a margin.